docs/includes/topic_hierarchical-port-binding.rst

```diff
@@ -0,0 +1,136 @@
+
+Use Case
+--------
+
+The most common use case for heirarchical port binding is an :term:`undercloud` deployment of a physical BIG-IP device or :term:`device service cluster` that processes traffic on networks dynamically created via SDN. When the F5 agent is configured with the name of a switch and the port(s) to which BIG-IP devices are connected, the LBaaSv2 driver discovers  Neutron networks in that switch's network segment. The driver provides the segmentation IDs of VLANs in the network segment to the F5 agent, which then dynamically creates the VLAN tags required to connect LBaaS services to the BIG-IPs.
+
+
+
+.. figure:: ../media/lbaasv2_hierarchical-port-binding.png
+    :alt: F5 LBaaSv2 Hierarchical Port Binding
+
+    F5 LBaaSv2 Hierarchical Port Binding
+
+
+Prerequisites
+-------------
+
+- Licensed, operational BIG-IP :term:`device` or :term:`device cluster`.
+
+- Operational OpenStack cloud (|openstack| release).
+
+- Administrator access to both BIG-IP device(s) and OpenStack cloud.
+
+- Software Defined Networking (SDN) service.
+
+- F5 :ref:`agent <agent:home>` and :ref:`service provider driver <install-f5-lbaasv2-driver>` installed on the Neutron controller and all other hosts for which you want to provision LBaaS services.
+
+- Knowledge of `OpenStack Networking <http://docs.openstack.org/liberty/networking-guide/>`_ concepts.
+
+- Knowledge of BIG-IP `system configuration`_, `local traffic management`_, & `device service clustering`_.
+
+
+Caveats
+-------
+
+- In release v |release| of the F5 LBaaSv2 driver and agent, ``VLAN`` is the only supported ML2 network type.
+
+- Each F5 agent managing a BIG-IP :term:`device service cluster` must have the same ``f5_network_segment_physical_network`` setting. [#]_
+
+-  If multiple F5 agents are managing the same environment, all of the agents must use the same binding settings (in other words, either the default global segmentation bindings or hierarchical port binding). [#]_
+
+
+Configuration
+-------------
+
+1. Edit the :ref:`Agent Configuration File`:
+
+.. code-block:: text
+
+    $ sudo emacs /etc/neutron/services/f5/f5-openstack-agent.ini
+
+
+2. Configure the heirarchical port binding settings in the :ref:`L2/L3 Segmentation Modes` section of the config file.
+
+.. table:: Hierarchical Port Binding settings
+
+    +--------------------------------------+-------------------------------------------+---------------+
+    | Setting                              | Description                               | Default Value |
+    +======================================+===========================================+===============+
+    |f5_network_segment_physical_network   || The name of the network segment in which | None          |
```

Comment:
Please elaborate in the description for this first setting.  The setting MUST be commented out to run in "traditional mode" (not disconnected).  The setting MUST be uncommented and set to a valid network name to run in "disconnected mode".  We can also fix the comment block in the ini file, but I'm trusting customers to use these new docs to understand the feature.